playkit/src/main/java/com/kaltura/playkit/drm/DrmCallback.java

```diff
@@ -1,43 +1,88 @@
 package com.kaltura.playkit.drm;
 
 import android.net.Uri;
+import android.text.TextUtils;
 
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.common.collect.ImmutableMap;
+import com.kaltura.android.exoplayer2.C;
 import com.kaltura.android.exoplayer2.drm.ExoMediaDrm;
 import com.kaltura.android.exoplayer2.drm.HttpMediaDrmCallback;
 import com.kaltura.android.exoplayer2.drm.MediaDrmCallback;
 import com.kaltura.android.exoplayer2.drm.MediaDrmCallbackException;
+import com.kaltura.android.exoplayer2.upstream.DataSource;
+import com.kaltura.android.exoplayer2.upstream.DataSourceInputStream;
+import com.kaltura.android.exoplayer2.upstream.DataSpec;
 import com.kaltura.android.exoplayer2.upstream.HttpDataSource;
+import com.kaltura.android.exoplayer2.upstream.StatsDataSource;
+import com.kaltura.android.exoplayer2.util.Util;
 import com.kaltura.playkit.PKLog;
 import com.kaltura.playkit.PKRequestParams;
+import com.kaltura.playkit.player.MediaSupport;
+
+import org.json.JSONObject;
 
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import java.util.UUID;
 
 public class DrmCallback implements MediaDrmCallback {
     private static final PKLog log = PKLog.get("DrmCallback");
-
     private final HttpDataSource.Factory dataSourceFactory;
     private final PKRequestParams.Adapter adapter;
     private HttpMediaDrmCallback callback;
+    private final Map<String, String> postBodyMap = new HashMap<>();
+    private String licenseUrl;
```

Comment:
can we flip the lines?
   private String licenseUrl;
   private final Map<String, String> postBodyMap = new HashMap<>();

Reply:
@GouravSna